x-pack/legacy/plugins/siem/public/components/timeline/styles.tsx

```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLoadingSpinner } from '@elastic/eui';
+import { rgba } from 'polished';
+import styled, { css } from 'styled-components';
+
+/**
+ * OFFSET PIXEL VALUES
+ */
+
+export const OFFSET_SCROLLBAR = 17;
+
+/**
+ * EVENTS TABLE
+ */
+
+export const EventsTable = styled.div.attrs({
```

Comment:
The effort to make the vertical scroll bar within the Timeline always-visible is very much appreciated @MichaelMarcialis !